### PR TITLE
[zh-cn]Update source-ip.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ The output is similar to this:
 -->
 输出类似于：
 ```
-NAME                   STATUS     ROLES    AGE     VERSION
+NAME                           STATUS     ROLES    AGE     VERSION
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/services/source-ip.md:137]: Line 137 is not necessary to change.

File: content/zh-cn/docs/tutorials/services/source-ip.md [hunk 2]
```diff
@@ -430,7 +429,7 @@ service/loadbalancer exposed
 Print out the IP addresses of the Service:
 -->
 打印 Service 的 IP 地址：
-```shell
+```console
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/services/source-ip.md:432]: shell should be fine.

**Review by tengqm — COMMENTED** [review]
/approve

**Pull request merged**